src/sensai/util/metadata.py

```diff
@@ -0,0 +1,64 @@
+from functools import lru_cache
```

Comment:
I think a better name for this module would be `pandas`, as it contains pandas-related utils (much like `utils.pickle` contains pickle-related utils).

Reply:
Renamed it